docs/05-reference/examples.mdx

```diff
@@ -13,4 +13,5 @@ into your project.
 | [Livepeer Web API Proxy](https://github.com/victorges/livepeer-web-api-proxy) | This Proxy extends the ability to limit access to the Livepeer Studio API for the web so that you can implement your own authentication method (e.g., if security or billing are of concern). You will be able to allow for creating streams directly from the browser without exposing access to an entire account via your API key. You can maintain, i.e. iterate and update, on the live deployment leveraging [fly.io](https://fly.io/) features. |
 | [Livepeer NFT Gate](https://github.com/livepeer/livepeer-nft-gate-example)    | This code provides steps to build a Livepeer NFT Gate app so that you can validate that, from a specific contract, a viewer owns an NFT in order to view your livestream.                                                                                                                                                                                                                                                                              |
-| [Livepeer Studio Sample App](https://github.com/livepeer/studio-sample-app)           | This sample application demonstrates the capabilities of what to expect when building with Livepeer Studio. When building a project with Livepeer Studio, there are examples provided for features such as On Demand or the Video Player. These examples provide a more in-depth approach using the API or plug-and-play with the SDK.                                                                                                                                                                                     |
\ No newline at end of file
+| [Livepeer Studio Sample App](https://github.com/livepeer/studio-sample-app)   | This sample application demonstrates the capabilities of what to expect when building with Livepeer Studio. When building a project with Livepeer Studio, there are examples provided for features such as On Demand or the Video Player. These examples provide a more in-depth approach using the API or plug-and-play with the SDK.                                                                                                                 |
+| [Livepeer Studio React Native Sample App](https://github.com/livepeer/livepeer-demo-react-native) | This is a sample mobile application for Android and iOS, built with React-Native. It showcases how to broadcaster and view a Livepeer Studio video directly in a mobile app.                                                                                                                                                                                                                                                     |
```

Comment:
Some typos and superfluous language here. Can we update to the following?

"This example application demonstrates how to add live video experiences to an application build with React Native using the Livepeer API."

Reply:
Let's also change "Livepeer Studio React Native Sample App" to "React Native Example"